flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormatTest.java

```diff
@@ -133,6 +133,32 @@ public void testExceptionOnInvalidType() throws IOException {
 		jdbcOutputFormat.writeRecord(row);
 	}
 
+	@Test(expected = RuntimeException.class)
+	public void testCastFloatToDoubleType() throws IOException {
+		jdbcOutputFormat = JDBCOutputFormat.buildJDBCOutputFormat()
+			.setDrivername(DRIVER_CLASS)
+			.setDBUrl(DB_URL)
+			.setQuery(String.format(INSERT_TEMPLATE, INPUT_TABLE))
+			.setSqlTypes(new int[] {
+				Types.INTEGER,
+				Types.VARCHAR,
+				Types.VARCHAR,
+				Types.FLOAT,
+				Types.VARCHAR})
+			.finish();
+		jdbcOutputFormat.open(0, 1);
+
+		Row row = new Row(5);
+		row.setField(0, 4);
+		row.setField(1, "hello");
+		row.setField(2, "world");
+		row.setField(3, 0.99f); // jdbcOutputFormat will cast float to double
```

Comment:
I usually avoid trailing comments (it's already pretty self-explanatory from your test function name

Reply:
agree